docs/api/en/core/Object3D.html

```diff
@@ -166,6 +166,11 @@ <h3>[property:Vector3 scale]</h3>
 		The object's local scale. Default is [page:Vector3]( 1, 1, 1 ).
 		</p>
 
+		<h3>[property:Boolean twoPassTransparentRendering]</h3>
+		<p>
+			Whether or not double-sided, transparent objects should be rendered with two passes or not. Default is `true`.
```

Comment:
But if it is one-sided, it will be rendered in one pass in any case, correct? So the description should be like so:
```Should the object be rendered in two passes if its material is transparent and double-sided. Setting the property to `true` in this case produces a better image, but makes rendering about two times slower. Default is `true`.```

Reply:
> But if it is one-sided, it will be rendered in one pass in any case, correct?

Yes

> Setting the property to `true` in this case produces a better image, but makes rendering about two times slower

I don't think terms like "better" or "two times slower are appropriate. Maybe @WestLangley can help with the formulation.